Sources/CodeEditSourceEditor/Controller/TextViewController.swift

```diff
@@ -306,6 +342,15 @@ public class TextViewController: NSViewController {
             delegate: self
         )
 
+        // Initialize guide view
+        self.guideView = ReformattingGuideView(column: reformatAtColumn, isVisible: showReformattingGuide, theme: theme)
+        if let guideView = self.guideView {
+            guideView.wantsLayer = true
+            guideView.layer?.zPosition = 1
+            textView.addSubview(guideView)
+            guideView.updatePosition(in: textView)
+        }
+
```

Comment:
You mentioned seeing how we can get rid of the lint error. I'd suggest removing this `if` section to either `loadView` or a new `styleGuideView` (that should also be in `loadView`). Also, I'd suggest moving the `wantsLayer` and `zPosition` to the guide view's initializer. Those properties should be handled by the view itself.

Reply:
Done